network-api/networkapi/wagtailpages/migrations/0160_alter_articlepage_body.py

```diff
@@ -0,0 +1,1285 @@
+# Generated by Django 4.2.15 on 2024-09-09 17:37
+
+import django.core.validators
+import wagtail.blocks
+import wagtail.contrib.table_block.blocks
+import wagtail.documents.blocks
+import wagtail.embeds.blocks
+import wagtail.fields
+import wagtail.images.blocks
+from django.db import migrations
+from wagtail.blocks.migrations.migrate_operation import MigrateStreamData
+
+import networkapi.wagtailpages.pagemodels.customblocks.articles
+import networkapi.wagtailpages.validators
+from networkapi.utility.migration.operations import AlterStreamChildBlockDataOperation
+
+
+def migrate_tableblock(source_block):
+    """Copy old table data to new table a TableBlock."""
+    return {
+        **source_block,
+        "value": {
+            "table": source_block["value"],
+            "wide": True,
```

Comment:
```suggestion
            "wide": False,
```

Hi @robdivincenzo!

After taking a look at the [related issue](https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10163) for this PR, it appears that design has requested that the "wide" option be set to `False` as the default. 

Can we please update the default here and also in the model definition?

Thanks!

Reply:
Left a comment on the other thread 🙂